yalu102/jailbreak.m

```diff
@@ -847,8 +847,13 @@ void exploit(void* btn, mach_port_t pt, uint64_t kernbase, uint64_t allprocs)
         {
             __block pid_t pd = 0;
             NSString* execpath = [[NSString stringWithUTF8String:pt]  stringByDeletingLastPathComponent];
-            
-            
+
+            if (dlsym(NULL, "system") == NULL) {
+
+                NSLog(@"function ´system´ not found. Using ´popen´ instead.");
+                #define system(x) popen(x,"r")
```

Comment:
This define will be always set, even if system symbol is found. preprocessor vs runtime.

Just use function that wraps system() with a different name and assign the pointer to it.